R/render.R

```diff
@@ -840,6 +840,8 @@ render <- function(input,
 
     perf_timer_start("pre-processor")
 
+    output_format <- merge_output_format_dependencies(output_format, knit_meta)
+
```

Comment:
I think we should make this conditional. This is safer than just relying on `merge_output_format_dependencies` to do nothing IMO. More clear also from `render.R` file 

```suggestion
    if (has_dependencies(knit_meta, "output_format_dependency")) {
        output_format <- merge_output_format_dependencies(output_format, knit_meta)
    }
```

What do you think ?

Reply:
Yes I understand why you feel that. But thank you :)